gee/landpks/src/main.py

```diff
@@ -479,7 +479,7 @@ def greenness_trend(year_start, year_end, geojson, lang, gc_client, metadata):
             .visualize(**{'palette': ['black'], 'opacity': 1})]) \
             .mosaic()
     
-    # Reproject ndvi mean image so it can retrieve data from every latitute 
+    # Reproject ndvi mean image so it can retrieve data from every latitude 
     # without deforming the aoi bounding box
     map_trnd_mosaic = map_trnd_mosaic.reproject(**({'crs':'EPSG:3857','scale':30}))
```

Comment:
And I hadn't noticed it before, but can you fix this ** notation also?

Reply:
kwarg notations removed for all reprojections